src/test/java/org/broadinstitute/consent/http/db/UserRoleDAOTest.java

```diff
@@ -44,19 +44,19 @@ private void resetUserRoleTable() {
         // TODO: In concept, I don't like this approach and hope to migrate all db tests to be
         // independant of one another
         userRoleDAO.deleteAllUserRoles();
-        userRoleDAO.insertSingleUserRole(2, 1, false);
-        userRoleDAO.insertSingleUserRole(5, 1, false);
+        userRoleDAO.insertSingleUserRole(2, 1);
```

Comment:
Having constants for these arguments would help make this more readable.

Reply:
👍

Reply:
We actually have constants for them that were added in after these tests were updated. Will make this cleaner.